frontend/common/dispatcher/app-actions.js

```diff
@@ -37,6 +37,12 @@ const AppActions = Object.assign({}, require('./base/_app-actions'), {
       projectId,
     })
   },
+  getEnv(environmentId) {
```

Comment:
Is there a reason why we need to get the environment this way? It adds to tech debt, ideally it would be an action in useEnvironment.ts